OneCommunityGlobal/HighestGoodNetworkApp
# Remove ability for anyone but Owners/Admins to see the green check by…

# Description

 (PRIORITY HIGH) Jae: Remove ability for anyone but Owners/Admins to resolve tasks
    1. Only Owners/Admins should be able to see the green check by tasks in the Tasks tab of the Dashboard and be able to click  it to resolve/make those tasks go away

![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49083865/fdf36264-3ae3-4627-8862-f5d6b25f1ffc)

## Mainly changes explained:
    1. Added a constant ‘isAllowedToResolveTasks’ to check if the current user role is allowed to resolve tasks.
    2. Added conditional rendering to the green check, so that the component is only displayed when the user is allowed to resolve tasks
## How to test:
    1. check into current branch
    2. do `npm run start:local` to run this PR locally
    4. log as (admin/owner) role user and select the Tasks tab
    5. check if the green ckeck is displayed.
    6. log in as a user with a role other than admin or owner, who has a task assigned to them, and check if the green check appears again.

## Screenshots of changes:

### Befor:

![befor](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49083865/79e2dd49-6040-4c67-b0ab-78e0478a7aed)

### After:

![after](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49083865/bfd9a5f2-a89c-4dcd-b983-6c9169744c08)
